client/client.go

```diff
@@ -564,6 +559,11 @@ type (
 		SignalWithStartWorkflow(ctx context.Context, workflowID string, signalName string, signalArg interface{},
 			options StartWorkflowOptions, workflow interface{}, workflowArgs ...interface{}) (WorkflowRun, error)
 
+		// NewWithStartWorkflowOperation returns a WithStartWorkflowOperation to perform Update-with-Start.
+		// Returns an error if the WorkflowIDConflictPolicy is not set, or if the workflow or arguments are invalid.
+		// NOTE: Experimental
+		NewWithStartWorkflowOperation(options StartWorkflowOptions, workflow any, args ...interface{}) (*WithStartWorkflowOperation, error)
```

Comment:
Why have one parameter take `any` and other take `interface{}`? They are alias's, but we should be consistent. I think the Go SDK generally takes `interface{}`

Reply:
It was unintended that they weren't consistent. Switched to `interface{}`. I personally prefer to use `any` but agree, we should be consistent thrughout the SDK.